STORM-IRIT/scripts/convert_txt.py

```diff
@@ -0,0 +1,9 @@
+import sys
+import os
+import numpy as np
+
+if __name__ == '__main__':
+    arr = np.loadtxt(sys.argv[1], delimiter="\n")
```

Comment:
I think the format used by authors is different (they label planes from `0` and mark `-1` for unsegmented, while we use `0` for unsegmented and mark planes from `1`)

Reply:
fixed